### ISSUE TITLE
Question about `pi0_base` generalization

### ISSUE THREAD

**Issue by LNTH** [Physical-Intelligence/openpi#285]
Hi,  

I'm experimenting with the pretrained `pi0_base` weights on AlohaSim. Specifically, I want to test the model on **AlohaSim Task 2: "AlohaInsertion-v0"**.  

From my understanding, the tutorial in `examples/aloha_sim/README.md` loads the `pi0_base` weights and `norm_stats.json` from `lerobot/aloha_sim_transfer_cube_human`.  

I have the following questions:  
- Can the `pi0_base` weights be used for the `AlohaInsertion` task by modifying `default_prompt="Transfer cube"`?
  - If so, could you suggest an appropriate prompt? None of my attempts have worked so far.
- How is `norm_stats.json` computed for `aloha_sim_transfer_cube_human`? If I want to use `AlohaInsertion`, should I recompute the normalization stats?  

Thanks in advance for your help!

**Comment by uzhilinsky** [comment]
> Can the pi0_base weights be used for the AlohaInsertion task by modifying...

Not out of the box, you'll need to fine tune the base model similar to what we do with `pi0_aloha_sim`. If you fine-tune, the exact problem won't really matter, just use something that describes the task in a reasonable way.

> How is norm_stats.json computed for aloha_sim_transfer_cube_human?

This is done by running:
```bash
uv run scripts/compute_norm_stats.py --config-name=pi0_aloha_sim
```
More info [here](https://github.com/Physical-Intelligence/openpi?tab=readme-ov-file#2-defining-training-configs-and-running-training).

You can also pull the norm stats from a different checkpoint (see `AssetsConfig` in the configs).

**Comment by LNTH** [comment]
Yeah, I just realized that I got confused between pi0_aloha and pi0_aloha_sim. Thanks @uzhilinsky
